.yamllint.yml

```diff
@@ -1,5 +1,8 @@
 ---
 extends: "default"
+ignore: |
+  /nautobot_circuit_maintenance/fixtures/
```

Comment:
So the only files we're linting at this point are the docker-compose files? To my mind those aren't very useful to lint, but linting the test fixtures would be more useful.

Reply:
The fixtures were taken directly from the DB dump export (this is the reason they don't have quotes) and used just for testing, so I didn't consider them as critical but maybe I'm missing something.

Reply:
I guess it comes back the question of what is the purpose of linting our YAML files at all. :-) If it's to ensure that they're stylistically consistent and avoid basic errors, it should be applied to everything, IMO.